integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/synapseconfig/MaxOpenConnections/nhttp/axis2.xml

```diff
@@ -105,7 +100,7 @@
 
     <!-- This deployment interceptor will be called whenever before a module is initialized or -->
     <!-- service is deployed -->
-    <listener class="org.wso2.carbon.core.deployment.DeploymentInterceptor"/>
+    <!--<listener class="org.wso2.carbon.core.deployment.DeploymentInterceptor"/>-->
```

Comment:
Shall remove this instead of commenting out?

Reply:
+1. If this is not applicable at all to the MI, we shouldn't have it at all.

Reply:
Fixed in: https://github.com/wso2/micro-integrator/pull/590